client/components/PostForm.jsx

```diff
@@ -0,0 +1,53 @@
+import styles from './PostForm.module.css';
+import { useState } from 'react';
+import axios from 'axios';
+
+const PostForm = () => {
+    const [text, setText] = useState('');
+    const [placeholder, setPlaceHolder] = useState(true);
+    const [len, setLen] = useState(145);
+
+
+    const postResponse = async () => {
+
+        const formData = { message: text }
+
+        if (len >= 0 && len !== 145) {
+            try {
+                const res = await axios.post('/api/home/post', formData);
+                console.log(res);
+            } catch (err) { console.log(err); }
+        }
+    };
+
+    const getValue = () => {
+
+        let value = document?.getElementsByClassName(styles.span)[0].innerText
```

Comment:
is this the only way to get value from the span ? we can't store value inside a state instead of doing document.bla blabla ?

Reply:
like we did with input , onChange and setting event value to a state ?